server/src/utils/sendgrid.js

```diff
@@ -40,6 +40,10 @@ const sendWelcomeMail = (user, token) => {
   })
 }
 
+const sendInfoMail = async (user) => {
```

Comment:
you have sendWelcome function (that I'm not sure works) But we eventually need only one of them